src/templates/pycontw-2020/contents/zh/guidelines.html

```diff
@@ -0,0 +1,102 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}PyCon Taiwan COVID-19 防疫守則{% endblock title %}
+
+{% block content %}
+
+<section class="pill-title alt">
+	<h1>PyCon Taiwan COVID-19 防疫守則</h1>
+</section>
+<h5 style="text-align:right;">(CC BY-SA 3.0 TW)<br/>最後更新時間：2020-08-03</h5>
+<hr>
+<p>
+	因應 2019 年新型冠狀病毒 (COVID-19) 疫情，「PyCon TW 主辦團隊」(以下簡稱「我們」) 參照我國中央流行疫情指揮中心的防疫措施，發布臺灣 Python 年會 (以下簡稱「PyCon TW」)
+	實體活動防疫守則。請您與我們一同遵守，保護自己與他人的健康。此防疫守則將視疫情發展與我國中央疫情疫情指揮中心指示有所變動，所有更動將不另通知，請在會前持續回來關心。最終版預計在 2020 年 08 月 20 日發布。
+</p>
+<hr>
+
+<h4>防疫要點</h4>
+
+<ol>
```

Comment:
nit: It's better to have line spacing between the items in the ordered list. You can customize the style for it (ref: [page.scss](https://github.com/pycontw/pycon.tw/blob/master/src/static/pycontw-2020/styles/page.scss))